src/scripts/main.js

```diff
@@ -1,3 +1,40 @@
 'use strict';
 
-// write code here
+function conversion(value) {
+  return +value
+    .split('').slice(1).join('').split(',').join('');
+}
+
+addEventListener('click', e => {
+  const sort = document.querySelector('thead').children[0].children;
+  const cheak = document.querySelector('tfoot');
```

Comment:
Consider using a more descriptive variable name than 'cheak'. For example, 'tableFooter'.